buildifier/integration_test.sh

```diff
@@ -0,0 +1,15 @@
+INPUT="def f(): pass"  # formatted differently in build and bzl modes
+BUILD_OUTPUT="def f(): pass"
+BZL_OUTPUT="def f():\n  pass"
+
+mkdir test
+echo -e "$INPUT" > test/BUILD
+echo -e "$INPUT" > test/test.bzl
+
+$1 --type=auto test/*
+
+echo -e "$BUILD_OUTPUT" > test/BUILD.golden
+echo -e "$BZL_OUTPUT" > test/test.bzl.golden
+
+diff test/BUILD test/BUILD.golden
+diff test/test.bzl test/test.bzl.golden
```

Comment:
Are you sure the test fails if the diff is incorrect?
Don't you need to return non-zero?

Reply:
Yes, diff returns a non-zero code and the whole script returns a non-zero code in that case.
If you just edit one of the variables, `bazel test //...` will crash.

Reply:
But the return value of the previous diff is ignored?

Reply:
I suggest using `set -e` at the top (https://stackoverflow.com/a/1379904/482260)

Reply:
You're right, added `set -e` to crash on the first error.